### ISSUE TITLE
operator_usage_whitespace rule false positive for unwrapping of the optional optional

### ISSUE THREAD

**Issue by madyanov** [realm/SwiftLint#2728]
- [x] Updated SwiftLint to the latest version (**0.31.0**)
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

SwiftLint thinks that `??` is a nil coalescing operator, while it's a unwrapping of the optional optional.

```swift
func success(for item: Item) {
    item.successHandler??() // <- triggers operator_usage_whitespace, bc there's no spaces around ??, but item.successHandler is an optional optional
}
```

Would be nice to make `operator_usage_whitespace` rule configurable so you can exclude specific operators (`??`). #1013

**Comment by cdstamper** [comment]
@jpsim it may be helpful to see some context (particularly the definition of item.successHandler), could you post that?

**Comment by marcelofabri** [comment]
Fixed in #3962
